static/CSS/style.css

```
@@ -747,6 +747,37 @@ input[type="file"] {
+  color: #fff;
+  border: none;
+  border-radius: 50px;
+  padding: 10px 18px;
+  font-size: 14px;
+  font-weight: bold !important;
+  cursor: pointer;
+  text-decoration: none;
+  margin-left: 5px;
+  display: inline-block;
+  text-align: center;
+  transition: all 0.2sease-in-out;
```

Biome (1.9.4)

[error] Unexpected unknown unit: sease-in-out

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)